test/visual-diff/visual-tests.json

```diff
@@ -101,6 +101,24 @@
         ".comic-amp-font-loaded",
         ".comic-amp-bold-font-loaded"
       ]
+    },
+    {
+      "url": "examples/visual-tests/amp-story/amp-story.amp.html",
+      "name": "amp-story",
+      "loading_complete_css": [
+        ".i-amphtml-story-standalone"
+      ]
+    },
+    {
+      "url": "examples/visual-tests/amp-story.bookend/amp-story.bookend.amp.html",
+      "name": "amp-story bookend",
+      "loading_incomplete_css": [
+        "amp-story:not(.i-amphtml-story-bookend-active)"
```

Comment:
Is this a valid CSS element name? The Percy build seems to think this continues to remain on the page.

Reply:
It is a valid CSS selector, but seems like it's not waiting for the `loading_complete_css` classes to be present (or the `loading_incomplete_css` classes to be absent)

Reply:
The default wait time is 2 seconds, but can be overridden. Is that insufficient for your test page?

https://github.com/teamcapybara/capybara/blob/master/README.md#asynchronous-javascript-ajax-and-friends

Reply:
Should basically be instantaneous.... so I'm not sure what's wrong 😞

Reply:
I've now created a PR that lets you enable experiments in visual tests: #12110

When I ran this PR after enabling the experiments in debug mode (`gulp visual-diff --debug`), I noticed this javascript error in your test page. Is this expected? Note that the browser on which these tests are being run is phantomjs, so I'm not sure if that's the source of the error.

```
144 [Resources] task failed: amp-story#1#L amp-story#1 TypeError: undefined is not a function (evaluating 'activePriorSibling.matches('amp-story-page')')
```

Reply:
This is not an expected error, but I agree it sounds like it might be something about the environment.  I'll see if I can debug locally.

Reply:
FYI, the snapshots generated during the failed build look like this: https://percy.io/ampproject/amphtml/builds/413802?snapshot=16945273